README.md

```diff
@@ -1 +1,27 @@
-Poort8.Ishare.Service
\ No newline at end of file
+Poort8.Ishare.Service
+
+
+# Testing
```

Comment:
In the context of git repos _testing_ is usually referred to as unit/integration testing. Here it means (for the lack of a better term) _playing_ with the endpoint.